[08-Jul-25 20:50:01 - INFO] - StringGen - @Devil_ai_robot Started.
[08-Jul-25 20:59:00 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-Jul-25 20:59:00 - INFO] - StringGen - Stopping String Gen Bot...
[08-Jul-25 21:02:06 - INFO] - StringGen - @Devil_ai_robot Started.
[08-Jul-25 21:02:10 - ERROR] - pyrogram.dispatcher - Peer id invalid: -1002681025373
Traceback (most recent call last):
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\methods\advanced\resolve_peer.py", line 62, in resolve_peer
    return await self.storage.get_peer_by_id(peer_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\storage\sqlite_storage.py", line 142, in get_peer_by_id
    raise KeyError(f"ID not found: {peer_id}")
KeyError: 'ID not found: -1002681025373'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyromod\listen\listen.py", line 237, in resolve_future
    await self.registered_handler(client, message, *args)
  File "E:\repo\auto-responder\StringGen\modules\start.py", line 20, in f_start
    await Anony.send_message(
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\methods\messages\send_message.py", line 128, in send_message
    peer=await self.resolve_peer(chat_id),
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\methods\advanced\resolve_peer.py", line 89, in resolve_peer
    peer_type = utils.get_peer_type(peer_id)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\utils.py", line 246, in get_peer_type
    raise ValueError(f"Peer id invalid: {peer_id}")
ValueError: Peer id invalid: -1002681025373
[08-Jul-25 21:03:21 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-Jul-25 21:03:21 - INFO] - StringGen - Stopping String Gen Bot...
[08-Jul-25 21:03:47 - INFO] - StringGen - @Devil_ai_robot Started.
[08-Jul-25 21:03:54 - ERROR] - pyrogram.dispatcher - Peer id invalid: -1002316223394
Traceback (most recent call last):
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\methods\advanced\resolve_peer.py", line 62, in resolve_peer
    return await self.storage.get_peer_by_id(peer_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\storage\sqlite_storage.py", line 142, in get_peer_by_id
    raise KeyError(f"ID not found: {peer_id}")
KeyError: 'ID not found: -1002316223394'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyromod\listen\listen.py", line 237, in resolve_future
    await self.registered_handler(client, message, *args)
  File "E:\repo\auto-responder\StringGen\modules\start.py", line 20, in f_start
    await Anony.send_message(
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\methods\messages\send_message.py", line 128, in send_message
    peer=await self.resolve_peer(chat_id),
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\methods\advanced\resolve_peer.py", line 89, in resolve_peer
    peer_type = utils.get_peer_type(peer_id)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\utils.py", line 246, in get_peer_type
    raise ValueError(f"Peer id invalid: {peer_id}")
ValueError: Peer id invalid: -1002316223394
[08-Jul-25 21:06:11 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-Jul-25 21:06:11 - INFO] - StringGen - Stopping String Gen Bot...
[08-Jul-25 21:07:43 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:07:46 - INFO] - StringGen - @Devil_ai_robot Started.
[08-Jul-25 21:07:52 - ERROR] - StringGen - Failed to send log message: Peer id invalid: -1002316223394
[08-Jul-25 21:09:18 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:09:21 - INFO] - StringGen - @Devil_ai_robot Started.
[08-Jul-25 21:09:26 - ERROR] - StringGen - Failed to send log message: Peer id invalid: -1002316223394
[08-Jul-25 21:09:56 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:10:00 - INFO] - StringGen - @Devil_ai_robot Started.
[08-Jul-25 21:10:00 - ERROR] - StringGen - Failed to send log message: Peer id invalid: -1002316223394
[08-Jul-25 21:11:40 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:12:15 - INFO] - StringGen - @Devil_ai_robot Started.
[08-Jul-25 21:12:15 - ERROR] - StringGen - Failed to send log message: Peer id invalid: -1002316223394
[08-Jul-25 21:12:23 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:12:26 - INFO] - StringGen - @Devil_ai_robot Started.
[08-Jul-25 21:14:00 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:14:01 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:14:04 - INFO] - StringGen - @Devil_ai_robot Started.
[08-Jul-25 21:14:05 - INFO] - StringGen - @Devil_ai_robot Started.
[08-Jul-25 21:15:31 - ERROR] - StringGen - Failed to send log message: Peer id invalid: -1002316223394
[08-Jul-25 21:15:31 - ERROR] - StringGen - Failed to send log message: Peer id invalid: -1002316223394
[08-Jul-25 21:17:29 - ERROR] - StringGen - Telegram says: [420 FLOOD_WAIT_X] - A wait of 1947 seconds is required (caused by "auth.ImportBotAuthorization")
[08-Jul-25 21:17:29 - ERROR] - StringGen - Failed to send log message: Peer id invalid: -1002316223394
[08-Jul-25 21:17:29 - ERROR] - StringGen - Failed to send log message: Peer id invalid: -1002316223394
[08-Jul-25 21:19:35 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:19:39 - INFO] - StringGen - @urr_wissh_bot Started.
[08-Jul-25 21:19:44 - ERROR] - StringGen - Failed to send log message: Peer id invalid: -1002316223394
[08-Jul-25 21:19:55 - INFO] - httpx - HTTP Request: POST https://batbin.me/api/v2/paste "HTTP/1.1 415 Unsupported Media Type"
[08-Jul-25 21:21:16 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-Jul-25 21:21:16 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-Jul-25 21:21:16 - INFO] - StringGen - Stopping String Gen Bot...
[08-Jul-25 21:21:16 - INFO] - StringGen - Stopping String Gen Bot...
[08-Jul-25 21:21:40 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:21:42 - INFO] - StringGen - @urr_wissh_bot Started.
[08-Jul-25 21:23:09 - INFO] - httpx - HTTP Request: POST https://batbin.me/api/v2/paste "HTTP/1.1 415 Unsupported Media Type"
[08-Jul-25 21:23:09 - ERROR] - StringGen - Batbin API error: 415 - 
[08-Jul-25 21:33:11 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-Jul-25 21:33:11 - INFO] - StringGen - Stopping String Gen Bot...
[08-Jul-25 21:38:18 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:38:24 - INFO] - StringGen - @urr_wissh_bot Started.
[08-Jul-25 21:38:29 - ERROR] - StringGen - Failed to send log message: Peer id invalid: -1002316223394
[08-Jul-25 21:39:12 - ERROR] - StringGen - Failed to send log message: Peer id invalid: -1002316223394
[08-Jul-25 21:42:58 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-Jul-25 21:42:58 - INFO] - StringGen - Stopping String Gen Bot...
[08-Jul-25 21:43:18 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:43:20 - INFO] - StringGen - @urr_wissh_bot Started.
[08-Jul-25 21:47:11 - ERROR] - pyrogram.dispatcher - 'list' object has no attribute 'write'
Traceback (most recent call last):
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\dispatcher.py", line 240, in handler_worker
    await handler.callback(self.client, *args)
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyromod\listen\listen.py", line 293, in resolve_future
    await self.registered_handler(client, query, *args)
  File "E:\repo\auto-responder\StringGen\modules\callbacks.py", line 96, in back_to_start_callback
    await callback_query.edit_message_text(
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\types\bots_and_keyboards\callback_query.py", line 200, in edit_message_text
    return await self._client.edit_message_text(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\methods\messages\edit_message_text.py", line 87, in edit_message_text
    reply_markup=await reply_markup.write(self) if reply_markup else None,
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\repo\auto-responder\venv\Lib\site-packages\pyrogram\types\bots_and_keyboards\inline_keyboard_markup.py", line 63, in write
    buttons.append(await b.write(client))
                         ^^^^^^^
AttributeError: 'list' object has no attribute 'write'
[08-Jul-25 21:50:00 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-Jul-25 21:50:00 - INFO] - StringGen - Stopping String Gen Bot...
[08-Jul-25 21:50:18 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:51:14 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:51:17 - INFO] - StringGen - @urr_wissh_bot Started.
[08-Jul-25 21:51:24 - INFO] - httpx - HTTP Request: POST https://batbin.me/api/v2/paste "HTTP/1.1 415 Unsupported Media Type"
[08-Jul-25 21:51:24 - ERROR] - StringGen - Batbin API error: 415 - 
[08-Jul-25 21:51:30 - ERROR] - StringGen - Failed to send log message: Peer id invalid: -1002316223394
[08-Jul-25 21:52:13 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:52:16 - INFO] - StringGen - @urr_wissh_bot Started.
[08-Jul-25 21:54:52 - ERROR] - StringGen - Failed to send log message: Peer id invalid: -1002316223394
[08-Jul-25 21:54:53 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:54:56 - INFO] - StringGen - @urr_wissh_bot Started.
[08-Jul-25 21:56:24 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-Jul-25 21:56:24 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-Jul-25 21:56:24 - INFO] - StringGen - Stopping String Gen Bot...
[08-Jul-25 21:56:24 - INFO] - StringGen - Stopping String Gen Bot...
[08-Jul-25 21:56:41 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 21:56:44 - INFO] - StringGen - @urr_wissh_bot Started.
[08-Jul-25 21:59:43 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-Jul-25 21:59:43 - INFO] - StringGen - Stopping String Gen Bot...
[08-Jul-25 22:00:01 - ERROR] - StringGen - Failed to verify logger group: Peer id invalid: -1002316223394
[08-Jul-25 22:00:04 - INFO] - StringGen - @urr_wissh_bot Started.
[08-Jul-25 22:00:05 - ERROR] - StringGen - Failed to send log message: Peer id invalid: -1002316223394
[08-Jul-25 22:05:15 - INFO] - root - Stop signal received (SIGINT). Exiting...
[08-Jul-25 22:05:15 - INFO] - StringGen - Stopping String Gen Bot...
[08-Jul-25 22:05:51 - ERROR] - StringGen - Telegram says: [420 FLOOD_WAIT_X] - A wait of 822 seconds is required (caused by "auth.ImportBotAuthorization")
[08-Jul-25 22:08:03 - ERROR] - StringGen - Error in bot startup: Telegram says: [420 FLOOD_WAIT_X] - A wait of 690 seconds is required (caused by "auth.ImportBotAuthorization")
[08-Jul-25 22:08:03 - ERROR] - StringGen - invalid literal for int() with base 10: 'A'
